### ISSUE TITLE
Error: Wave audio format id 3 is not supported

### ISSUE THREAD

**Issue by Fatihcrim** [echogarden-project/echogarden#47]
![Error](https://github.com/echogarden-project/echogarden/assets/166035817/728c2671-30fc-41ff-8c88-850ed314f90a)
The error is in the title. When I try to align anything, it gives this error. OS is Win 11. Please help me 😞

**Comment by rotemdan** [comment]
Thanks a lot for reporting.

Seems like the wave file has a sample format id of `3`. Based on the specifications:

![Screenshot_1](https://github.com/echogarden-project/echogarden/assets/8589488/8e880cad-7a5f-4a85-9158-78e1485ffd9b)

This means it is `WAVE_FORMAT_IEEE_FLOAT`

The wave format encoder/decoder should support it, but for whatever reason there's an apparent typo in the definition code that labels this format as `2`, and not `3`:

```ts
export enum SampleFormat {
	PCM = 1,
	Float = 2,
	Alaw = 6,
	Mulaw = 7,
}
```

Interesting it wasn't caught earlier. I guess floating point wave files aren't that common.

I wrote the wave encoder/decoder myself, so it is easy for me to fix.

Can you send me the particular `wav` file to reproduce the problem and ensure that is the actual issue / verify the problem is fixed?

**Comment by Fatihcrim** [comment]
https://www.mediafire.com/file/ch63b35821llhy7/Englishvery.wav/file
thank you 🙌

**Comment by rotemdan** [comment]
Thanks. I tested the file. It works now. The problem was just the typo in the code.

I'm currently working on a big release (`1.0.0`) with lots of new features and fixes, so it may take a few days until the fix is available.

(GitHub supports uploads by the way, you can just drag the file to the GitHub text editor and it will upload and include it in the message)

**Comment by Fatihcrim** [comment]
Thank you for responding so quickly. I tried to upload the file here, but it gave an error saying it does not support wav files, so I did this. 🫡

**Comment by rotemdan** [comment]
Should be fixed in the new release `v1.0.0`.

Let me know if you encounter any other issues.
